src/containers/FeedContainer.ts

```diff
@@ -1,29 +1,44 @@
 import { connect } from 'react-redux';
-import { AppState } from '../reducers';
+import { AppState, defaultLocalPosts } from '../reducers';
 import { StateProps, DispatchProps, FeedView } from '../components/FeedView';
 import { AsyncActions, Actions } from '../actions/Actions';
 import { Feed } from '../models/Feed';
 import { getFeedPosts, getYourPosts } from '../selectors/selectors';
 
 export const mapStateToProps = (state: AppState, ownProps: { navigation: any }): StateProps => {
     const feedUrl = ownProps.navigation.state.params.feedUrl;
-    const selectedFeeds = state.feeds.filter(feed => feed != null && feed.feedUrl === feedUrl);
-    // TODO: we should have here state.ownFeeds.find(feed => feed.url === feedUrl) != null,
-    // but author.uri is empty, and the feedUrl from navparams is from post.author.uri (not real feedUrl)
-    // For the same reason feeds/selectedFeeds is empty (errounously)
-    const isOwnFeed = state.author.uri === feedUrl;
+    const feedName = ownProps.navigation.state.params.name;
+
+    const isOwnFeed = feedName === state.author.name;
+    const isAssistantFeed = feedName === 'Felfele Assistant';
+    const hasOwnFeed = state.ownFeeds.length > 0;
+    const ownFeed = hasOwnFeed ? state.ownFeeds[0] : undefined;
+    const selectedFeeds = isOwnFeed
+        ? hasOwnFeed
```

Comment:
maybe this would be less ambiguous if we call this variable `isFeedSynchronized` or `isFeedSetUp` or...

Reply:
I don't know what feed setup means in the context of the codebase. I know what feed synchronization means in the context of the codebase and it's not what we are trying to express. So to me the suggested names are more ambiguous.

Reply:
Yes these are not good either, I was just looking forint the correct wording. hasRemoteCounterpart is a bit long.